subprojects/docs/src/docs/userguide/building_java_projects.adoc

```diff
@@ -335,13 +324,11 @@ The one and only packaging feature provided by the Java Plugin directly is a `ja
 
 If you want any other JAR or alternative archive built, you either have to apply an appropriate plugin or create the task manually. For example, if you want a task that generates a 'sources' JAR, define your own `Jar` task like so:
 
-=== Example: Defining a custom task to create a 'sources' JAR
-
-[source.multi-language-sample,groovy]
-.build.gradle
-----
-include::{samplesPath}/userguide/java/basic/build.gradle[tag=defining-sources-jar-task]
-----
+.Defining a custom task to create a 'sources' JAR
+====
+include::sample[dir="userguide/java/basic/groovy",files="build.gradle[tags=defining-sources-jar-task]"]
+include::sample[dir="userguide/java/basic/kotlin",files="build.gradle.kts[tags=defining-sources-jar-task]"]
+====
 
 See link:{groovyDslPath}/org.gradle.api.tasks.bundling.Jar.html[Jar] for more details on the configuration options available to you. And note that you need to use `classifier` rather than `appendix` here for correct publication of the JAR.
 
```

Comment:
I think we also missed a couple samples "Creating a fat JAR" and manifest ones below.

Reply:
Yep: see above.